subsys/bluetooth/host/classic/sdp.c

```diff
@@ -1746,11 +1746,89 @@ static int sdp_client_ss_search(struct bt_sdp_client *session,
 	return bt_sdp_send(&session->chan.chan, buf, BT_SDP_SVC_SEARCH_REQ, session->tid);
 }
 
+static uint16_t sdp_client_get_attribute_id_list_len(struct bt_sdp_attribute_id_list *ids)
+{
+	uint16_t len = 0;
+
+	if (ids == NULL || ids->count == 0) {
+		return sizeof(uint8_t) + sizeof(uint32_t);
+	}
+
+	for (size_t i = 0; i < ids->count; i++) {
+		if (ids->ranges[i].beginning == ids->ranges[i].ending) {
+			len += sizeof(uint8_t) + sizeof(uint16_t);
```

Comment:
My only suggestion is: changing `sizeof(uint8_t) + sizeof(uint16_t)` as one MACRO like:`#define BT_SDP_UINT16_DATA_SIZE (sizeof(uint8_t) + sizeof(uint16_t))`.

Reply:
OK.